### ISSUE TITLE
DxDateEdit Datepicker not open in correct position (when allow grouping) inside DisplayTemplate of Data Grid

### ISSUE THREAD

**Issue by mdosmaneng** [DevExpress/Blazor#80]
Hi,
I use `DxDateEdit-Nullable Date` inside `DisplayTemplate` of Blazor Data Grid. When I click to open datepicker, it does not open in correct position. I already tried with custom css to resolve this issue but does not work this. 

I attached this issue in the following project. Please! find the attachment.

[Sample.zip](https://github.com/DevExpress/Blazor/files/3998801/Sample.zip)

**Comment by zabelin-vladimir** [comment]
Hello @mdosmaneng 

We are aware of this issue and are working on it in the context of the [DataGrid for Blazor - The drop-down window is shown under the grid on editing a ComboBox column](https://www.devexpress.com/Support/Center/Question/Details/T810006/datagrid-for-blazor-the-drop-down-window-is-shown-under-the-grid-on-editing-a-combobox) SC ticket. 
Thank you for your sample project, we will test it after fixing this issue.

**Comment by zabelin-vladimir** [comment]
Hello @mdosmaneng 

We have fixed the issue from the [DataGrid for Blazor - The drop-down window is shown under the grid on editing a ComboBox column](https://supportcenter.devexpress.com/ticket/details/t810006/datagrid-for-blazor-the-drop-down-window-is-shown-under-the-grid-when-editing-a-combobox) SC ticket in [our latest release](https://github.com/DevExpress/Blazor/releases/tag/v19.2.1-beta).
I checked your project with this version and it works properly.  This release is available via the DevExpress NuGet server. Refer to [this page](https://nuget.devexpress.com/) to obtain your personal NuGet feed URL.
